KTLOSNaxxLoot.json

```diff
@@ -576,7 +576,7 @@
 			"wowID":"23048"
 		},
 		"Slayer's Crest":{
-			"prio":["Bendriller(crush)","Destain(crush)","Johnscar(sunnyD)","Sniped(crush)","EP/GP"],
```

Comment:
Wait, don't you have Phylactery and DFT?  This doesn't seem reasonable atm

Reply:
I think all the crush rogues on this list have that setup, but I'm not inclined to LC this item to a brand new rogue yet. Not sure if I should let this be EP/GP or LC'd to Destain in crush and Johnscar in SunnyD.